src/Symfony/Component/PropertyAccess/NamingStrategy/NamingStrategyInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\NamingStrategy;
+
+/**
+ * @author David Badura <d.a.badura@gmail.com>
+ */
+interface NamingStrategyInterface
+{
+    /**
+     * Returns all possible getters.
+     *
+     * @param string $class
+     * @param string $property
+     *
+     * @return array
+     */
+    public function getGetters(string $class, string $property): array;
+
+    /**
+     * Returns all possible setters.
+     *
+     * @param string $class
+     * @param string $property
```

Comment:
I think this PHPDocs has no sense: they just duplicates typehints. But we can better describe return: `@return string[]`.